src/Api/KeyManagement/Models/Response/PublicAccountKeysResponseModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Bit.Core.Models.Api;
+
+namespace Bit.Api.KeyManagement.Models.Response;
+
+/// <summary>
+/// This response model is used to return the public keys of a user, to any other registered user or entity on the server.
+/// It can contain public keys (signature/encryption), and proofs between the two. It does not contain (encrypted) private keys.
+/// </summary>
+public class PublicAccountKeysResponseModel : ResponseModel
+{
+    public PublicAccountKeysResponseModel(string verifyingKey, string publicKey, string publicKeyOwnershipSignature)
+        : base("userKeys")
```

Comment:
Since we already use `accountKeys` for private response model, it might be good to keep it consistent.
```suggestion
        : base("publicAccountKeys")
```